### PR TITLE
Corolla TSS2 INDI tuning

### PR DESCRIPTION
**Pull request by birdman6450**
corollatss2 tune, never ping pong, never sway, accurate, always smooth, and significantly improves lateral. #needs dbc's for corollatss2 and corollatss2 hybrid before merge. 

-->

### CODE REVIEW

**Comment by brianandyt** [comment]
I’ve watch birdman spend the last 8 months simply removing ping pong on straights. To making best Corolla ride I’ve had on openpilot

**Comment by majorwolf** [comment]
> I’ve watch birdman spend the last 8 months simply removing ping pong on straights. To making best Corolla ride I’ve had on openpilot

100% agree with this statement. He has worked hard to make a amazing tune and I fully endorse it.

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -1424,7 +1424,8 @@ class CAR:
   CAR.HIGHLANDERH_TSS2: dbc_dict('toyota_nodsu_hybrid_pt_generated', 'toyota_tss2_adas'),
   CAR.AVALON: dbc_dict('toyota_avalon_2017_pt_generated', 'toyota_adas'),
   CAR.RAV4_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
-  CAR.COROLLA_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
+  CAR.COROLLA_TSS2: dbc_dict('toyota_nodsu_pt_generated_corolla_tss2', 'toyota_tss2_adas'),
+  CAR.COROLLAH_TSS2: dbc_dict('toyota_nodsu_pt_generated_corollah_tss2', 'toyota_tss2_adas'),	', 'toyota_tss2_adas'),
```
Comment (unleashthemind) [inline selfdrive/car/toyota/values.py:1428]: I think this line may have some typos in it

Comment (SCshredder17) [reply selfdrive/car/toyota/values.py:1428]: Thanks i will double check

Comment (birdman6450) [reply selfdrive/car/toyota/values.py]: thank you. typos fixed

**Comment by birdman6450** [comment]
Screen Shot 2021-01-12 at 9 52 36 AM

**Comment by birdman6450** [comment]
Screenshot_20210113-010434_Discord

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (rav4kumar) [reply selfdrive/car/toyota/values.py]: inorder to change safety param you need to modify dbc. so its not typo its more

**Comment by SCshredder17** [comment]
Screen Shot 2021-01-14 at 10 26 50 PM

**Comment by birdman6450** [comment]
Screenshot_20210114-222156_Discord

**Comment by birdman6450** [comment]
Screenshot_20210115-175615_Discord

**Comment by birdman6450** [comment]
Screenshot_20210117-171120_Discord

**Comment by adeebshihadeh** [comment]
Any reason this is still a draft?

**Comment by adeebshihadeh** [comment]
Closing due to inactivity.

**Pull request closed without merging**